client-landing/src/components/MessageSnackbars.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { connect } from 'react-redux';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { shiftMessage } from '../actions';
+
+const styles = theme => ({
+  close: {
+    padding: theme.spacing.unit / 2,
+  },
+  messageSnackbar: {
+    paddingBottom: 60, // Get it above the footer
+  },
+});
+
+class MessageSnackbars extends React.Component {
```

Comment:
I have this exact component in `de-components` lib called `CyverseAnnouncer`. Do you want to give that a shot ?